index.html

```diff
@@ -0,0 +1,118 @@
+                    <p><a href="https://accounts.intuit.com/signup.html?offering_id=Intuit.ifs.mint&namespace_id=50000026&redirect_url=https%3A%2F%2Fmint.intuit.com%2Foverview.event%3Futm_medium%3Ddirect%26cta%3Dhero_sign_up_free_ProspectWeb%26ivid%3Da9b8df23-0d05-490a-a527-662b045fd445%26adobe_mc%3DMCMID%253D43778899234622625232808106119344280814%257CMCORGID%253D969430F0543F253D0A4C98C6%252540AdobeOrg%257CTS%253D1600358737%26ivid%3Da9b8df23-0d05-490a-a527-662b045fd445#" rel="noopener noreferrer" target="_blank">I forgot my User ID or Password</a></p>
+                </div>
+            </div>
+            
+            <div class="signin-btn">
+             <button type="button" name="sign-in" value="Sign In"> <i class="material-icons">lock</i> Sign In</button>
+            </div>
+        </div> 
+            </section>
+        </header>
+        
+
+        <section>
+            <div class="logo">
+                <img src="images/descarga1.png" alt="intuit-logo" >
+            </div>
+          <div class="logos">
+              <div class="column">
+                <img src="images/descarga2.png" alt="mint-logo">
+              </div>
+              <div class="column">
+                <img src="images/descarga3.png" alt="quicklink-logo">
+              </div>
+                <div class="column">
+                <img src="images/descarga4.png" alt="turbotax-logo">
+              </div>
+          </div>
+        </section>
+
+        <section class="sign-up">
+                <div class="form-size">
+                    <div class="form-header">
+                        <div>
+                            <h1>Create an Intuit account</h1>
+                        </div>
+                        <div>
+                            <h3>One account for everything Intuit, including Mint.</h3>
+                        </div>
+                        <div>
+                        <h3> <a href="https://accounts-help.intuit.com/app/intuit/1995107" rel="noopener noreferrer" target="_blank">Learn more</a></h3>
+                        </div>
+                    </div>
+               
+
+                <form>
+                <div class="element-style">
+                    <div>
+                        <label>Email address</label>
+                        <input type="email" name="email">
+                    </div>
+
+                    <div>
+                        <label>Phone (recommended)</label>
+                        <input type="tel" name="tel">
+                        <p>Standard call, messaging or data rates may apply.</p>
+                    </div>
+                    
+                    <div>
+                        <label>Password</label>
+                        <input type="password" name="password">
+                    </div>
+                    
+                   <div>
+                    <button type=button><i class="material-icons">lock</i> <span>Create Account</span></button>
+                   </div>
```

Comment:
[OPTIONAL]  Vertically align the lock and the text. You could use `vertical-align` to achieve this as they both are inline elements.